ros/src/models/chrono/ros_chrono/src/steering_controller.cpp

```diff
@@ -215,6 +224,7 @@ int main(int argc, char* argv[]) {
     double frict_coeff = 0, rest_coeff = 0, gear_ratios = 1; //Chrono Vehicle parameters
```

Comment:
is the  frict_coeff  supposed to be zero?

Reply:
No, it should not. It will be override at [line245](https://github.com/ChangYuHsuan/MAVs/blob/7ac18e1b287ceac5a00b9582904b9b9b0318fa76/ros/src/models/chrono/ros_chrono/src/steering_controller.cpp#L245)

Reply:
Sorry, I don't know how to show the code like you did.

Reply:
hmm, why do these need to be set here if they are being read off of ROSparams?

Reply:
I am looking at the pull request to make these comments

Reply:
Nevermind, it does not matter. This seems fine

Reply:
I had modify the code as [velocity_controller.cpp](https://github.com/ChangYuHsuan/MAVs/blob/master/ros/src/models/chrono/ros_chrono/src/velocity_controller.cpp) where friction coefficient and rest coefficient are not predefined , I think it might look more organized.